.github/workflows/sc-client-server-deb10.yml

```diff
@@ -87,14 +138,22 @@ jobs:
       uses: geekyeggo/delete-artifact@v2
       with:
         name: Server Image
-    - name: Start SAI-C client
+    
+    - name: Start SAI-C in client mode
```

Comment:
There are two SAI-C modes: standalone, client-server.
In client-server mode we start SAI-C client and SAI-C server. So, please remove "in" here and below.